src/app/amsterdam/contact/gegevens/page.tsx

```diff
@@ -32,47 +33,45 @@ function Question() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
         <Breadcrumb>
           <NextLink legacyBehavior passHref href="/amsterdam">
-            <Breadcrumb.Item href="/amsterdam">Home</Breadcrumb.Item>
+            <Breadcrumb.Link href="/amsterdam">Home</Breadcrumb.Link>
           </NextLink>
         </Breadcrumb>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Contact</Heading>
-          <Fieldset legend="Wat zijn uw contactgegevens?">
-            <Column gap="small">
-              <div>
-                <Paragraph>Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.</Paragraph>
-              </div>
-              <div>
+          <FieldSet legend="Wat zijn uw contactgegevens?">
+            <Column gap="small" style={{ clear: 'both' }}>
+              <Paragraph>Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.</Paragraph>
+              <Field>
                 <Label htmlFor="voorletters">Voorletters</Label>
                 <TextInput id="voorletters" name="voorletters" />
-              </div>
-              <div>
+              </Field>
+              <Field>
                 <Label htmlFor="achternaam">Achternaam</Label>
                 <TextInput id="achternaam" name="achternaam" />
-              </div>
-              <div>
+              </Field>
+              <Field>
                 <Label htmlFor="email">E-mail</Label>
                 <TextInput id="email" name="email" />
-              </div>
+              </Field>
               <Row as="div">
```

Comment:
`div` is default, no?

Reply:
What do you mean?

A `Field` is also responsible for the whitespace between label and text input.

Reply:
No, I mean `<Row as="div">` and `<Row>` renders the exact same thing I think